test/suites/drift/suite_test.go

```diff
@@ -719,6 +719,23 @@ var _ = Describe("Drift", func() {
 		env.EventuallyExpectNotFound(pod, node)
 		env.EventuallyExpectHealthyPodCount(selector, numPods)
 	})
+	It("should update NodeClass hash on the NodeClass and NodeClaim", func() {
```

Comment:
Can you be more descriptive here? This isn't just about validating that we are updating the hash on the NodeClass and the NodeClaim, this is about testing this new versioning mechanism and validating that we are doing the repropagation like we think that we are